src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -544,92 +544,92 @@ export function getEventContent(event) {
   switch (event.type) {
     case EVENT_TYPES.claimDecision:
       return {
-        title: 'VBA sent the original claim decision to you',
+        title: 'VA sent you a claim decision',
         description: '',
       };
     case EVENT_TYPES.nod:
       return {
-        title: 'VBA received your Notice of Disagreement',
+        title: 'VA received your Notice of Disagreement',
         description: '',
       };
     case EVENT_TYPES.droHearing:
-      return {
+      return { // TODO: does this need to be updated?
         title: 'DRO Hearing',
         description: '',
       };
     case EVENT_TYPES.fieldGrant:
       return {
-        title: 'Field grant',
+        title: 'VA granted one or more issues',
         description: '',
       };
     case EVENT_TYPES.soc:
       return {
-        title: 'VBA prepared a Statement of the Case (SOC)',
+        title: 'VA sent you a Statement of the Case',
         description: '',
       };
     case EVENT_TYPES.form9:
       return {
-        title: 'Form 9 received',
+        title: 'VA received your Form 9',
         description: '',
       };
     case EVENT_TYPES.ssoc:
       return {
-        title: 'Supplemental Statement of the Case',
+        title: 'VA sent you a Supplemental Statement of the Case',
         description: '',
       };
     case EVENT_TYPES.certified:
       return {
-        title: 'The Board received your appeal',
+        title: 'Your appeal was transferred to the Board of Veterans’ Appeals',
         description: '',
       };
     case EVENT_TYPES.hearingHeld:
       return {
-        title: 'Your hearing was held at the regional office',
+        title: 'You attended a hearing with a Veterans Law Judge',
         description: '',
       };
     case EVENT_TYPES.hearingCancelled:
-      return {
+      return { // TODO: does this need to be updated?
```

Comment:
@gnakm should this also be updated?

Reply:
This can be removed

Reply:
Thanks @gnakm, this event (hearingCancelled) has been removed.